tutorials/cloud-iot-rtdp/bin/create_temp_alert_store.py

```diff
@@ -0,0 +1,12 @@
+import os
+from google.cloud import datastore
+
+client = datastore.Client(os.environ['PROJECT'])
```

Comment:
We usually use GCLOUD_PROJECT or GOOGLE_PROJECT_ID for this environment variable.